src/analysis/return_value.rs

```diff
@@ -23,7 +23,13 @@ pub fn analyze(
     used_types: &mut Vec<String>,
     imports: &mut Imports,
 ) -> Info {
-    let typ = override_string_type_return(env, func.ret.typ, configured_functions);
+    let typ = match configured_functions.iter()
+                                              .filter_map(|f| f.ret.type_name.as_ref())
+                                              .map(|typ| env.library.find_type(0, typ))
+                                              .next() {
+        Some(Some(typ)) => typ,
```

Comment:
To remove double some, please use .and_then, and maybe `if let` instead match

Reply:
I don't see how `.and_then` would improve the situation. Another `.filter_map` could though but then we'd look to the next arg and I don't want that so I think the only improvement possible is to use `if let` as you suggested.

Reply:
I thought about something like this:
```
    let typ = configured_functions.iter()
        .filter_map(|f| f.ret.type_name.as_ref())
        .next()
        .and_then(|type_name| env.library.find_type(0, type_name))
        .unwrap_or_else(|| override_string_type_return(env, func.ret.typ, configured_functions));
```

Reply:
Oh indeed! Didn't think about it this way...